### PR TITLE
/matches page

### PR DESCRIPTION
**Pull request by DMills94**
SQL only returns one match, ideally would return an array of matches but not sure how that works. I have a list of matches from the SQLlite db you provided me but not sure how to access

Goals of this page eventually would include filters by hero to see their match history

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -67,26 +67,6 @@
         "qs": "6.7.0",
         "raw-body": "2.4.0",
         "type-is": "~1.6.17"
-      },
```
Comment (omorgan7) [inline package-lock.json:70]: can you remove changes to package-lock.json? not 100% sure if they should be in the history.

File: database.js [hunk 2]
```diff
@@ -46,6 +46,19 @@ exports.validPlayers = async function(db, ids) {
     return await db.allAsync("SELECT * FROM player_table where id32 IN " + queryString, ids.flat())
 }
 
+exports.fetchMatches = async function(db) {
+    const matches = await db.allAsync("SELECT id, duration, winner, timestamp FROM match_table ORDER BY id DESC")
+    
+    for (let i = 0; i < matches.length; i++)  {
+        const matchId = matches[i].id
+        const heroes = await db.allAsync("SELECT hero_name, game_team FROM match_player_table WHERE match_id = ?", matchId)
+        matches[i].rad_heroes = heroes.filter(hero => hero.game_team === 2).map(hero => hero.hero_name)
```
Comment (omorgan7) [inline database.js]: think there's a utilities.RADIANT but doesn't really matter

File: utilities.js [hunk 3]
```diff
@@ -58,11 +58,21 @@ exports.equals = function(x, y) {
...
+    const minutes = Math.floor(remainingSeconds / 60)
+    const seconds = String(remainingSeconds - minutes * 60).padStart(2, "0")
 
-  exports.secondsToMinuteSeconds = function(duration) {
-      const minutes = Math.round(duration / 60)
-      const seconds = Math.round(duration % 60)
+    return {
+        hours,
+        minutes: padTime(minutes),
+        seconds: padTime(seconds)
```
Comment (omorgan7) [inline utilities.js:74]: think seconds gets padded twice

**Review by omorgan7 — CHANGES_REQUESTED** [review]
Looks good, thank you for doing this, couple of minor things, i don't think there should be any changes to package-lock as we weren't updating any dependencies in this PR? or have i misunderstood that

File: package-lock.json (hunk 1 above)
Comment (DMills94) [reply package-lock.json:70]: Yeah they should be, this is your version control and should always be up to date for if others are to clone the repo themselves

File: utilities.js (hunk 3 above)
Comment (DMills94) [reply utilities.js:74]: You're right i forgot to remove it from `L69`, will update

**Pull request merged**